app/models/order.rb

```diff
@@ -6,6 +6,8 @@ class Order < ApplicationRecord
 
   validates :name, :address, :email, presence: true
 
+  scope :by_date, -> (from: DateTime.now.at_beginning_of_day, to: DateTime.now) { where created_at: from..to }
```

Comment:
read diff about now vs current
why are we not doing end_of_day in to?
use beginning_of_day

Reply:
done